src/Application/Components/ClickCanvas/ClickCanvas.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import { fabric } from 'fabric-webpack';
+import numeral from 'numeral';
+
+const ClickCanvas = React.createClass({
+  propTypes: {
+    cash: React.PropTypes.number.isRequired,
+    income: React.PropTypes.number.isRequired,
+    onClick: React.PropTypes.func.isRequired,
+    image: React.PropTypes.string.isRequired,
+  },
+
+  images: {
+    penny: 'images/penny.gif',
+  },
+
+  getInitialState() {
+    return {
+      cash: 0,
+      income: 0,
+    }
+  },
+
+  _createText(x, y, color, text) {
```

Comment:
usually methods that create / modify things on the DOM (like returning jsx in normal components) are called `renderThing`